src/Resolution.ts

```diff
@@ -18,11 +18,12 @@ import {
   SourceConfig,
   Web3Version0Provider,
   Web3Version1Provider,
+  ZnsSupportedNetworks,
 } from './types/publicTypes';
 import ResolutionError, {ResolutionErrorCode} from './errors/resolutionError';
 import DnsUtils from './utils/DnsUtils';
 import {findNamingServiceName, signedInfuraLink} from './utils';
-import {Eip1993Factories} from './utils/Eip1993Factories';
+import {Eip1193Factories} from './utils/Eip1193Factories';
```

Comment:
@sudoryan Please don't rename this since it's a breaking change. Just add additional export alias as Vlad mentioned above

Reply:
In index.ts

```typescript
export {Eip1993Factories as Eip1193Factories} from './utils/Eip1993Factories';
export {Eip1993Factories as ProviderFactories} from './utils/Eip1993Factories';
```